app/controllers/experiments_controller.rb

```diff
@@ -369,11 +371,53 @@ def create
         replication_level: [:optional, :security_default, :integer, :positive],
         execution_time_constraint: [:optional, :security_default, :integer, :positive],
         parameter_constraints: [:optional, :security_json],
-        type: [:optional, :security_default]
+        type: [:optional, :security_default],
+        workers_scaling: [:optional] # TODO boolean validator
```

Comment:
To TODO oznacza sprawdzanie czy parametr jest booleanem, nie validator zwracający boolean.